Library/Formula/node.rb

```diff
@@ -1,8 +1,9 @@
 class Node < Formula
   desc "Platform built on the V8 JavaScript runtime to build network applications"
   homepage "https://nodejs.org/"
-  url "https://nodejs.org/dist/v5.7.0/node-v5.7.0.tar.gz"
-  sha256 "2338b46a2f45fbb747089c66931f62555f25a5928511d3a43bbb3a39dcded2d8"
+  url "https://nodejs.org/dist/v5.7.1/node-v5.7.1.tar.gz"
+  mirror "https://github.com/nodejs/node/archive/v5.7.1.tar.gz"
+  sha256 "dc6ad2f7c5ac0b87db5b20731c42a1c4fc22edb6a5da22ca93fe9ffd3eac95ce"
```

Comment:
CI disagrees:
```
==> Downloading https://nodejs.org/dist/v5.7.1/node-v5.7.1.tar.gz
Warning: Formula reports different SHA256: dc6ad2f7c5ac0b87db5b20731c42a1c4fc22edb6a5da22ca93fe9ffd3eac95ce
Downloaded to: /Library/Caches/Homebrew/node-5.7.1.tar.gz
SHA1: bd7825efe70b5f82b4a34df17f9f1826d490213d
SHA256: ecd4956335c6b86f1b72fd203abd9a833717e6800edb12208d0e3ce625330198
```

Your checksum matches the automatically generated GitHub tarball, which likely isn't the release tarball Node uses.

It'd be nice to move over to:
```
https://nodejs.org/dist/v5.7.1/node-v5.7.1.tar.xz
```
to be honest because the download size is considerably smaller thanks to better compression.